cli/smartcontract/contract_test.go

```diff
@@ -358,8 +361,13 @@ func TestContractDeployWithData(t *testing.T) {
 		}
 		e.In.WriteString(testcli.ValidatorPass + "\r")
 		e.Run(t, cmd...)
+		var tx *transaction.Transaction
+		if await {
+			tx, _ = e.CheckAwaitableTxPersisted(t)
+		} else {
+			tx, _ = e.CheckTxPersisted(t, "Sent invocation transaction ")
```

Comment:
While we're here, remove `Sent invocation transaction`, it's a remnant from old CLI.